program_transformers/src/asset_upserts.rs

```diff
@@ -95,6 +96,11 @@ pub struct AssetMetadataAccountColumns {
     pub royalty_amount: i32,
     pub asset_data: Option<Vec<u8>>,
     pub slot_updated_metadata_account: u64,
+    pub plugins: Option<Value>,
```

Comment:
Might be cleaner to store in a different table?

Reply:
Yeah I could do that, its probably better then prefixing them with `mpl_core_` in the main `asset` table.

Reply:
Tough balance between normalization and query speed. Right now though the model is complex the read times are quick because its as simple index lookup. I'd like to keep that read speed.

The interface on the asset maps the plugins to a specific protocol if another pluggable asset type is creating in the future.

Reply:
I am also happy keeping the new columns in the asset table. I do think it its better for the query speed.

Reply:
I did draft it up using separate `mpl_core` table here: https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/pull/180

As you guys have pointed out it will add the extra queries so we will skip it.